src/components/movieCard/MovieCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet, View, Text, TouchableOpacity, Image } from 'react-native';
+
+import styles from '@components/movieCard/style';
+import { memo } from 'react';
```

Comment:
keep imports in order

"
basic imports - react, 3rd party lib, etc

components

assets - globalStyles, utils, services

./style.ts

"

Also you can import neighboring files directly - as in here `./style`